### PR TITLE
Cleanup hack/*.py scripts and add a pylint job to keep them clean

### PR DESCRIPTION
**Pull request by petr-muller**

### CODE REVIEW

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml [hunk 1]
```diff
@@ -15,7 +15,11 @@ presubmits:
         image: registry.svc.ci.openshift.org/ci/promrules:latest
         imagePullPolicy: Always
         name: ""
-        resources: {}
+        resources:
+          limits:
+            cpu: 500m
```
Comment (stevekuznetsov) [inline ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: We're certain of these values for jobs that run ci-op, but I'm not absolutely certain about these jobs running other code. Once we change this if we see jobs goiing over the limit we can adjust

File: hack/validate-ci-operator-config-filename.py [hunk 2]
```diff
@@ -23,29 +23,24 @@ def main():
     errors = []
 
     if not os.path.isdir(args.config_dir):
-        errors.append(
-            "[ERROR] {} directory doesn't exist.".format(args.config_dir))
+        errors.append(f"[ERROR] {args.config_dir} directory doesn't exist.")
```
Comment (stevekuznetsov) [inline hack/validate-ci-operator-config-filename.py]: Wow, I've never seen this before. Leave it to Python to invent another way to format strings...

Comment (petr-muller) [reply hack/validate-ci-operator-config-filename.py]: Hehe. Yeah, this is 3.6+ stuff, but I have checked we are using 3.7 to run this, so I thought we might use it, it's nicely readable.

File: ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml (hunk 1 above)
Comment (petr-muller) [reply ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml]: Most of these seem to be small utilities, I think the limits should be fine.

**Comment by petr-muller** [comment]
> Seems like the format strings are inconsistent? Otherwise LGTM

What do you mean? That I only converted the formatting to literal interpolation in one script, not the other?

**Comment by stevekuznetsov** [comment]
Yes, that

**Pull request merged**